docs/labs/libvirt.md

```diff
@@ -98,6 +98,31 @@ You can change the parameters of the management network in the **addressing.mgmt
 * **\_network**: The *libvirt* network name (default: `vagrant-libvirt`)
 * **\_bridge**: The name of the underlying Linux bridge (default: `libvirt-mgmt`)
 
+## Starting Virtual Machines in Batches
+
+*vagrant-libvirt* plugin tries to start all the virtual machines specified in `Vagrantfile` in parallel. The resulting strain on CPU resources might cause VM boot failures in very large topologies. As a workaround, you can configure **libvirt** virtualization provider to execute a series of `vagrant up` commands to start the virtual machines in smaller batches:
+
+* Configure the batch size with **defaults.providers.libvirt.batch_size** parameter (an integer between 1 and 50)
+* Configure idle interval between batches (if needed) with **defaults.providers.libvirt.batch_interval** parameter (between 1 and 1000 seconds).
+
+Example:
+
+```
+provider: libvirt
+defaults.device: cumulus
+defaults.providers.libvirt.batch_size: 2
```

Comment:
I'd suggest a batch size default of a higher number, like 6. Remember people will usually start with your example, and if they use 2 it'll take ages to boot a 40 node topology!